### ISSUE TITLE
Bug or Mistake ? FilteringSelect / ComboBox not displayed correctly

### ISSUE THREAD

**Issue by ghost** [atoha/dbootstrap#46]
Hello,

I installed dbootstrap and everything is working fine except FilteringSelect / ComboBox and I can't understand why.

I followed the tutorial step by step and I still get this : 
![2013-04-30 11 50 44](https://f.cloud.github.com/assets/2699904/443346/ccd5ef66-b17b-11e2-852c-ef265dd80c3d.png)

However if I compile the demo, it's working fine

Do you have any idea why ?
Thanks

**Comment by martinpengellyphillips** [comment]
Hey! Sorry to hear you are having trouble getting this to work. 

Are you trying to integrate into your project's build or are you just building the theme and using that in your project?

Also, what Operating System and Browser (including version) are you using?

**Comment by ghost** [comment]
Hello,

There is a difference between the demo and default dojo, the html which is generated for the filtering select is different,
Your demo : 

``` html
<div class="dijitReset dijitRight dijitButtonNode dijitArrowButton dijitDownArrowButton dijitArrowButtonContainer" role="presentation" data-dojo-attach-point="_buttonNode, _popupStateNode" popupactive="true">
<span class="dijitReset dijitInputField dijitArrowButtonInner"></span>
</div>
```

Dojo :

``` html
<div class="dijitReset dijitRight dijitButtonNode dijitArrowButton dijitDownArrowButton dijitArrowButtonContainer" role="presentation" data-dojo-attach-point="_buttonNode, _popupStateNode" popupactive="true">
<input class="dijitReset dijitInputField dijitArrowButtonInner" aria-hidden="true" type="button" role="button presentation" readonly="readonly" tabindex="-1" type="text" value="▼ " />
</div>
```

dbootstrap is included in the project build, Dojo 1.8.3, bug on every browser.
Thanks

Edit : I found a solution, dbootstrap needs to be put first in require, don't know why but it works now !

**Comment by martinpengellyphillips** [comment]
Hey, glad to hear you figured it out.

I can probably make this clearer in the docs, but for the icons to work I have to replace some of the Dojo dom at runtime. This is what the icon_support.js file does, which is loaded as part of the require dbootstrap call. However, because it is patching some Dojo classes on the fly it **has** to be before any calls to your project code to ensure you get the patched class!
